### ISSUE TITLE
Max Number of Episode Per Podcast - Increase

### ISSUE THREAD

**Issue by sannitig** [gpodder/gpodder#835]
More of a much needed feature request than a bug - but still an extremely quick and easy fix.

Change the number of max number of episodes per podcast from 1000 to 10,000 or something of the like please. 

When searching certain podcasts with more than 1000 episodes, you only get the last 1000 and cannot view further back.

Preferences > Updating > Maximum number of episodes per podcast >XXXX

**Comment by auouymous** [comment]
See #692. You will have to use `edit config` and manually set `limit.episodes` to change it above or below the bounds.

@elelay I still think the control should have a larger min/max range but if not, could `limit.episodes` be mentioned next to the control? Something like "Exceed bounds with limit.episodes in Edit config".

**Comment by 3nk0** [comment]
Nice, someone asked the same question 20 hours before I wanted to ask. I tried this long time ago and it didn't work, still there has been a limit of 1000 or 2000. Now I used this way:

- Set `"episodes": 999999` in the gPodder configuration file `Settings.json`, using a text editor.
- Save and close the file. Start and quit gPodder without any other actions once.
- Now you can change the limit using the GUI.

I don't know the issue I described above has been solved in meantime and this way is nonsense now. In case not, you have to follow this.

**Comment by omgwtfbbqpls** [comment]
I changed the episode limit in Settings.json and the same number shows up in the GUI preferences, but the actual number of shown episodes isn't changing.

**Comment by auouymous** [comment]
@omgwtfbbqpls if it is a youtube channel, you need to use the youtube-dl extension to get more than 15 episodes.

**Comment by omgwtfbbqpls** [comment]
@auouymous It's not youtube channels. The[ JRE podcast ](http://joeroganexp.joerogan.libsynpro.com/rss)for example only shows episodes 1505-1334 even though I've set the limit to 10000.

**Comment by elelay** [comment]
You might have to unsubscribe and re-subscribe to the podcast.

**Comment by elelay** [comment]
See #907 for the re-subscribing issue.

**Comment by wealthistime** [comment]
I have this issue, but I cannot find any "Settings.JSON" file. Can someone tell me where it's at? I am viewing hidden files and searched the gPodder directory....I can't find any similar files either.

**Comment by auouymous** [comment]
@wealthistime Open preferences in gpodder and click the `Edit config` button at bottom of window. Then search for "limit", does the `limit.episodes` line have "999999" after it? If not, double click the number, change it, press enter and close the dialog.

**Comment by wealthistime** [comment]
Yes, it does have "999999", but i still only get 1,000 entries. The Podcast Addict app displays all of them, and I read on a different forum it s because Podcast Addict supports "pagination" whereas most apps don't. Still, I wish I could figure out a way to get gPodder to work 😕

**Comment by auouymous** [comment]
@wealthistime Gpodder supports pagination, what is the feed URL?

**Comment by wealthistime** [comment]
- I have three of them, all same result:
- https://www.spreaker.com/show/5679963/episodes/feed
- https://www.spreaker.com/show/5681567/episodes/feed
- https://www.spreaker.com/show/5681567/episodes/feed

Definitely owe you a beer either way, thanks so much for the help...really wanting this to work!

**Comment by auouymous** [comment]
The feed only has 1000 episodes in it and doesn't contain a `rel=next` link, which is how the atom format handles pagination. Maybe podcast addict has custom support for spreaker feeds and looks elsewhere to paginate them.

**Comment by wealthistime** [comment]
Thanks, that's the only thing i can figure as well...
